generators/ed25519.py

```diff
@@ -1533,7 +1533,7 @@ def _into_key_data(key_prefix, test_vectors):
     account.address = program_id
     account.owner = program_owner
     instr_ctx.accounts.extend([account])
-    instr_ctx.instr_accounts.extend([invoke_pb.InstrAcct()])
```

Comment:
Why was this removed?

Reply:
it's no longer needed. an instruction can now have 0 accounts
(the tx must have 1+ account, in this case the program)